Purpose: Correct startpar SEE ALSO section
Author:  Roger Leigh
Fixes:   Missing and unavailable references
Status:  Not submitted upstream

--- a/startpar/startpar.8	2012-04-09 20:16:32.101074559 +0100
+++ b/startpar/startpar.8	2012-04-09 20:17:06.077627790 +0100
@@ -114,9 +114,8 @@
 .I /etc/init.d/.depend.stop
 
 .SH SEE ALSO
-.BR init.d (7),
-.BR insserv (8),
-.BR startproc (8).
+.BR init (8)
+.BR insserv (8).
 
 .SH COPYRIGHT
 2003,2004 SuSE Linux AG, Nuernberg, Germany.
